src/components/Arrays.jsx

```diff
@@ -0,0 +1,74 @@
+  { id: 11, question: "This person's father was Enoch, and has the longest lifespan of Bible record: 969 years.", answer: "Methuselah" },
+  { id: 12, question: "Preach the word; be at it urgently in favorable times and ___ times. (2 Tim 4)", answer: "difficult" },
+              
+  // 600 row
+  { id: 13, question: "'For all things I have the strength through the one who gives me power.'", answer: "Paul" },
+  { id: 14, question: "Jehovah brought lasting judgment on this man's house because 'his sons are cursing God, but he has not rebuked them.'", answer: "Eli" },
+  { id: 15, question: "Jehovah reassured this prophet that his judgment will come at the right time: 'It will without fail come true. It will not be late!'", answer: "Habakkuk" },
+  { id: 16, question: "This king had the shortest reign (3 months and 10 days).", answer: "Jehoiachin" },
+  { id: 17, question: "Asenath bore Joseph two sons. This was the brother of Manasseh.", answer: "Ephraim" },
+  { id: 18, question: "He gave his parents honey to eat, and 'did not tell them he had scraped [it] out of the carcass of a ___.' (Judges 14)", answer: "Lion" },
+
+  // 800 row
+  { id: 19, question: "'Do not imagine that because you are in the king's household you are any more likely to escape than all the other Jews.'", answer: "Mordecai" },
+  { id: 20, question: "This person dreamt of 'the sun and the moon and 11 stars bowing down to [him].'", answer: "Joseph" },
+  { id: 21, question: "Jehovah instructed this prophet to endure spousal infidelity, and his marriage reflected Jehovah's loyal love and forgiveness towards Israel.", answer: "Hosea" },
+  { id: 22, question: "These two kings have the same name, were brothers-in-law, and ruled at the same time.", answer: "Jehoram" },
+  { id: 23, question: "This person's name meant 'One Seizing the Heel; Supplanter', and his brother's name meant 'Hairy'.", answer: "Jacob" },
+  { id: 24, question: "All of you clothe yourselves with ___ toward one another. (1 Peter 5)", answer: "humility" },
+
+  // 1000 row
+  { id: 25, question: "'Alas, O Sovereign Lord Jehovah! I do not know how to speak, for I am just a boy.'", answer: "Jeremiah" },
+  { id: 26, question: "This is what the promised Seed was called, which means 'He Whose It Is; He To Whom It Belongs'.", answer: "Shiloh" },
+  { id: 27, question: "This prophet may have been a great-great-grandson of King Hezekiah and a relative of King Josiah.", answer: "Zephaniah" },
+  { id: 28, question: "This is approximately how old Josiah was when Zebidah gave birth to his second son, Jehoiakim. (Insight book)", answer: "14" },
+  { id: 29, question: "Two Jacobs in the Bible both had a son with this name.", answer: "Joseph" },
+  { id: 30, question: "Abraham named that place ___. This is why it is still said today: 'In the mountain of Jehovah it will be provided.' (Gen 22)", answer: "Jehovah-jireh" }
+];
+
+export const listHeaders = categoryArr.map(header => 
```

Comment:
listHeaders is a component. It should be defined as a function:

```suggestion
export function ListHeaders() { 
  return categoryArr.map(header => ({
      <th key...>
        {header.category}
      </th>
    });
}
```

You can also choose to place it in its own file so it's easier to find.